### PR TITLE
openpiv

### PR DESCRIPTION
**Pull request by alexlib**

### CODE REVIEW

File: recipes/openpiv/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - scipy
+    - cython
+    - progressbar
```
Comment (jakirkham) [inline recipes/openpiv/meta.yaml]: Aren't these also needed at install?

File: recipes/openpiv/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - openpiv
+
+about:
+  home: https://github.com/openpiv/openpiv-python
+  license: GPL
```
Comment (jakirkham) [inline recipes/openpiv/meta.yaml]: What version of GPL (e.g. `GPL 3`)?

File: recipes/openpiv/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{% set version = "0.20" %}
+
+package:
+  name: openpiv
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/alexlib/openpiv-python.git
```
Comment (jakirkham) [inline recipes/openpiv/meta.yaml]: We prefer not to clone from version control. Please use a tarball instead and add a checksum (preferably `sha256`) like so.

```
  url: https://github.com/alexlib/openpiv-python/archive/{{ version }}.tar.gz
  sha256: <checksum>
```

Comment (jakirkham) [inline recipes/openpiv/meta.yaml]: Also, what is the reason for using a fork? We prefer to use the original source if possible. If we require patches, those can be applied (depending on their purpose). However, those patches should be submitted upstream.

File: recipes/openpiv/meta.yaml (hunk 2 above)
Comment (jakirkham) [inline recipes/openpiv/meta.yaml]: Also please add the license file like so.

```yaml
  license_file: LICENSE.txt
```

**Comment by jakirkham** [comment]
Something we can help with here, @alexlib?

**Comment by alexlib** [comment]
no, thanks. trying in another pull request #1337

**Pull request closed without merging**